screen-pop-app/screen-pop/src/grantLogin.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useRef } from "react";
+import axios from "axios";
+import "./App.css";
+import { Channel } from "./notification";
+import { ConfigData,  ConversationManager } from "./ConversationManager";
+
+const CLIENT_ID = "81a7fe1e-497b-4b99-aef0-eb67192750e9";
```

Comment:
instead of defining these in multiple files, use a `.env` file in the react app's root dir and specify them as `REACT_APP_CLIENT_ID` and `REACT_APP_ENVIRONMENT`. https://create-react-app.dev/docs/adding-custom-environment-variables/